types/index.ts

```diff
@@ -2,11 +2,13 @@ export type jobProps = {
     id: string;
     roleTitle?: string;
     location?: string;
-    minSalary?: number;
```

Comment:
Again not sure if we want to optional capture a salary range.

Double check with @Hangout2698

Reply:
confirmed with @Hangout2698 that salary in string is okay, if company wants range then can specify the values with dash "10-20"